modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -37,17 +37,20 @@ spec:
       networks:
         - name: default
           pod: {}
-        - name: bridge-net <1>
+        - name: bridge-net <2>
           multus:
-            networkName: <network-namespace/a-bridge-network> <2>
+            networkName:  <network-namespace>/<a-bridge-network> <3>
```

Comment:
There are 2 spaces between `networkName:` and ` <network-namespace>`. Remove one please.